FlightJournal.Web/FlightJournal.Web.csproj

```diff
@@ -138,8 +145,7 @@
     <Reference Include="System.Web.Entity" />
     <Reference Include="System.Web.ApplicationServices" />
     <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Core" />
```

Comment:
- [ ] Was System.Core not used... ?

Reply:
Hmmm. Guess not. Don't remember removing this explicitly. Might have been due to the update to .Net Framework 4.7.2